salon_creator/lib/firebase/database.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:salon_creator/models/salon.dart';
+
+final salonsRef = FirebaseFirestore.instance.collection('salons').withConverter(
+      fromFirestore: (snapshot, _) => Salon.fromMap(
+        snapshot.data(),
+      ),
+      toFirestore: (salon, _) => salon.toMap(),
+    );
+
+void addSalonsToDatabase(Salon salon) async {
```

Comment:
I like the idea of having a single file to do all db related operations. Nice work!
Since we're adding only salon to db at a time via this function, I felt like this is a more appropriate name for the function:
```suggestion
void addSalonToDatabase(Salon salon) async {
```
Also, bonus points if you can make this to a class like **DbHandler** which contains methods like **addSalon** etc:
```
DbHandler dbHandler = DbHandler();
dbHandler.addSalon(salon);
Salon salon = dbHandler.getSalon(salonName);
```